pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -106,6 +106,9 @@ type ScrapeConfigSpec struct {
 	// Authorization header to use on every scrape request.
 	// +optional
 	Authorization *v1.SafeAuthorization `json:"authorization,omitempty"`
+	// TLS configuration to use on every scrape request
+	// +optional
+	TLSConfig *v1.TLSConfig `json:"tlsConfig,omitempty"`
```

Comment:
I'd stick with SafeTLSConfig for now, especially since #5618 might simplify TLS configuration.

```suggestion
	TLSConfig *v1.SafeTLSConfig `json:"tlsConfig,omitempty"`
```

Reply:
done